docs/ur/README.md

```diff
@@ -0,0 +1,351 @@
+# Style guide Urdu (ur)
+
+ <!-- toc -->
+
+## Introduction
+
+This style guide provides instruction for localizing to the Urdu locale. It contains rules that are both defined by Mozilla and Mozilla's localization communities on how to best translate text in Mozilla products, websites, and other projects. By following these rules, the translator has a better chance of producing a high quality translation that represents Mozilla values and culture. This style guide is to be used in coordination with the [General Mozilla L10n Style Guide](../mozilla_general/). We welcome your feedback, questions and concerns regarding the Style Guide.
+
+### Team Details
+
+* Wiki: [https://wiki.mozilla.org/L10n:Teams:ur](https://wiki.mozilla.org/L10n:Teams:ur)
+* Pontoon: [https://pontoon.mozilla.org/ur/](https://pontoon.mozilla.org/ur/)
+* Telegram: [join](https://t.me/joinchat/Ci1LBhM7ccHcG3tFYQG6bQ)
+
+## General Considerations
+
+The localized text should be as if it was originally written in Urdu. It has to be accurate, correct and clear. To achieve that try to avoid wordiness and word-by-word translation.
+* **Accuracy**: As a rule all English text needs to be translated. In some cases though, text can be omitted or transliterated; there should be a specific reason for that. The translated text will correctly reflect product functionality.
+
+```
+
+Example:- Sync
```

Comment:
'Sync' with a capital letter is a brand name, and shouldn't be translated. I suggest making this 'sync' instead.

Reply:
Sure. Got it.